content/en/apps/tutorials/localizing-cht.md

```diff
@@ -0,0 +1,169 @@
+
+You should have a functioning [CHT instance with `cht-conf` installed locally]({{< ref "apps/tutorials/local-setup" >}}), completed a [project folder]({{< ref "apps/tutorials/local-setup#3-create-and-upload-a-blank-project" >}}) setup, and an [messages-sw.properties]({{< ref "apps/reference/translations" >}}) file.
+
+## Implementation Steps
+
+Create a new file in the 'translations/' folder called messages-sw.properties
+
+1. Localize CHT Elements to Swahili
+_**CHT Instance text**_ - for non-admin users, this is the text that falls under **Messages**, **Tasks**, **Reports**, **People** and **Targets**.
+To localize instance text to Swahili, change the default system language to `Swahili`.
+
+![configuration](change-system-language.png)
+
+Use the instructions outlined [here](https://docs.communityhealthtoolkit.org/apps/reference/translations/#translations) to  change the instance language.
+
+
+After changing the instance language, the various elements will behave like this:
+_**Messages**_
+In Messages, the time counter text and text that shows the user how to navigate the tab changes.
+
+See below example for Swahili localization.
+
+![configuration](messages-tab-language.png)
+
+_**Tasks**_
+You can localize the task header by adding the appropriate translation in the `messages-sw.properties file.
+
+e.g to translate the below delivery task title to Swahili
+```
+{
+    name: 'anc-home-visit-delivery',
+    icon: 'icon-pregnancy',
+    title: 'task.anc.delivery.title',
+``` 
+
+Add the below code to the messages-sw.properties file
+```
+task.anc.delivery.title = Kazi ya Kujifungua
+```
+![configuration](localize-tasks.png)
+
+_**Reports**_
+You can localize the report field names by adding the appropriate translation in the `messages-{language-code}.properties` file.
+
+e.g to change the date of birth field to Swahili, in `messages-sw.properties` file, add this:
+
+```
+contact.type.date_of_birth = Siku ya Kuzaliwa
+```
+![configuration](localize-reports.png)
+
+
+e.g 
+
+_**People**_
+To localize the contact labels, add the appropriate translation in `messages-{language-code}.properties` file.
+
+e.g to change the chv name field to Swahili, in `messages-sw.properties` file, add this:
+
+```
+contact.type.district_hospital = Kituo Cha Afya
+```
+
+_**Targets**_
+You can localize the names of the targets by adding the appropriate translation in the `messages-{language-code}.properties file.
+
+e.g to change the `Number of fever cases managed` target to Swahili on the instance, adding the appropriate translation in the `messages-sw.properties` file. i.e
+
+```
+targets.fever_cases_managed.title = Idadi ya kesi za homa kusimamiwa
+```
+
+#### 2. App Forms
+To localize an app form to say the Kiswahili language, open the appropriate xlsx of the form and add a `label::sw` column which has the translation for the text. This will work in the `Survey` sheet or the `choices` sheet.
+
+![configuration](app-forms-localization.png)
```

Comment:
Let's also show the two form renderings from the app

Reply:
It's `label::sw`. Not sure what the question is referring to here since neither `label::id` nor `label::en` is in the doc.

Reply:
label::id is indonesian. so it's OK. I'd lost that context